src/content/app/genome-browser/components/browser-track-config/track-config-views/GeneTrackConfig.tsx

```diff
@@ -73,8 +73,10 @@ export const GeneTrackConfig = () => {
         <div className={styles.toggleWrapper}>
           <label>Transcript IDs</label>
           <SlideToggle
-            isOn={shouldShowTranscriptIDs}
-            onChange={() => updateShowTranscriptIds(!shouldShowTranscriptIDs)}
+            isOn={shouldShowTranscriptLabels}
+            onChange={() =>
+              updateShowTranscriptLabels(!shouldShowTranscriptLabels)
+            }
```

Comment:
I am a bit conflicted with this syntax that seems to be used more frequently in the code. Why do you prefer this (inline function) over using a function expression?

Reply:
haha that is true and you get so annoyed 😒   I will change it